host-interaction/registry/create-or-open-registry-key.yml

```diff
@@ -1,10 +1,10 @@
 rule:
   meta:
     name: create or open registry key
```

Comment:
analogous to `create or open file`, we have a lot more specific rules that use this (now library) rule, e.g. `set registry value` or `delete registry value`